mjpc/estimators/batch.cc

```diff
@@ -106,13 +109,19 @@ void Batch::Initialize(const mjModel* model) {
 
   // dimension
   int nq = model->nq, nv = model->nv, na = model->na;
-  int ntotal = nv * max_history_;
+  int nconfig = nv * max_history_;
```

Comment:
I understand that "nconfig" is referring to the word "configuration" as in "the pose of the body", but this is somewhat clashing with the common use of "config" meaning "settings".  It's not a big deal, but I'd also consider "nframes" or "npose".

Reply:
Renamed `nconfig` -> `nvel`.